readers/openapi.yml

```diff
@@ -140,6 +147,55 @@ components:
       schema:
         type: string
       required: false
+    Value:
+      name: value
+      description: SenML message value.
+      in: query
+      schema:
+        type: string
+      required: false
```

Comment:
This seems redundant. You have `V` param.

Reply:
It is, but it means that we support `v` or `value`. I did it following this filters: https://github.com/mainflux/mainflux/blob/master/readers/api/transport.go#L33

Reply:
Can you use `$ref` to reference Value instead of copy-pasting definition?

Reply:
The name is different. It's not really copy paste. Should I maybe remove `value` parameter and keep `v`, `vs`,  `vd` and `vb`?

Reply:
I agree. The input JSON is using `v`, so it makes sense to use it in the filter.